consensus/vbft/state_mgmt.go

```diff
@@ -428,27 +429,34 @@ func (self *StateMgr) checkStartSyncing(startBlkNum uint32, forceSync bool) erro
 
 // return 0 if consensus not reached yet
 func (self *StateMgr) getConsensusedCommittedBlockNum() (uint32, bool) {
-	C := int(self.server.config.C)
+	myCommitted := self.server.GetCommittedBlockNo()
+
+	commitedBlocksList := make([]uint32, 0, len(self.peers))
+	for _, p := range self.peers {
+		if p.committedBlockNum > myCommitted {
+			commitedBlocksList = append(commitedBlocksList, p.committedBlockNum)
+		}
+	}
 
+	sort.Slice(commitedBlocksList, func(i, j int) bool { return commitedBlocksList[i] > commitedBlocksList[j] })
+
```

Comment:
for simple, change to the following?

```
if len(committedBlocksList) <= C || committedBlocksList[C] < myCommitted { 
    return myCommitted, false 
} else {
    return  committedBlocksList[C], true
}
```

Reply:
@Honglei-Cong 
1. commitedBlockslist is a list of committed blocks.
len(committedBlocksList) - returns number of peers, so we can't compare it with the quorum value

2. committedBlocksList[C] - number of committed block of one of the peers, e.x. if quorum value is 2, than committedBlocksList[C] returns committed block of the third peer.

Reply:
Right.  After commitedBlockList was sorted by order of height, committedBlocksList[C] is the consensused block height?

Reply:
Yes, after commitedBlockList is sorted it has sens. I'll do these changes.